tutorials/getting_started/auto_overview.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: documentation
+title: Pages - Overview Page
+---
+
+# Automatically Generated Overview Tabs
+
+MainUI will automatically generate an Overview page (id:overview).
+This page will have four tabs.
+
+- Overview - user defined page.
+- Locations - populated based on the model showing your Items based on their physical location.
+- Equipment - populated based on the model showing your Items based on their Equipment type.
+- Properties - populated based on the model showing your Items based on their semantic Properties tag.
```

Comment:
I am still not sure about this section but have no better idea for now.
Maybe a little table would be more suitable.

Or we leave this short explanation out and write something like

"This page will ahve four tabs which are decribed below"

Reply:
I reworked it to just list the four without the explanatory sentence. I'll leave that detail to the sections.